posthog/tasks/usage_report.py

```diff
@@ -416,6 +417,40 @@ def get_teams_with_billable_enhanced_persons_event_count_in_period(
     return result
 
 
+@timed_log()
+@retry(tries=QUERY_RETRIES, delay=QUERY_RETRY_DELAY, backoff=QUERY_RETRY_BACKOFF)
+def get_teams_with_billable_anonymous_personfull_event_count_in_period(
+    begin: datetime, end: datetime, count_distinct: bool = False
+) -> list[tuple[int, int]]:
+    # anonymous events that are still personfull.
+    # count only unique events
+    # Duplicate events will be eventually removed by ClickHouse and likely came from our library or pipeline.
+    # We shouldn't bill for these. However counting unique events is more expensive, and likely to fail on longer time ranges.
+    # So, we count uniques in small time periods only, controlled by the count_distinct parameter.
+    if count_distinct:
+        # Uses the same expression as the one used to de-duplicate events on the merge tree:
+        # https://github.com/PostHog/posthog/blob/master/posthog/models/event/sql.py#L92
+        distinct_expression = "distinct toDate(timestamp), event, cityHash64(distinct_id), cityHash64(uuid)"
+    else:
+        distinct_expression = "1"
+
+    result = sync_execute(
+        f"""
+        SELECT team_id, count({distinct_expression}) as count
+        FROM events
+        WHERE timestamp between %(begin)s AND %(end)s
+            AND event != '$feature_flag_called' AND event NOT IN ('survey sent', 'survey shown', 'survey dismissed')
+            AND person_mode IN ('full', 'force_upgrade')
+            AND JSONExtractBool(properties, '$is_identified') = 0
```

Comment:
@robbie-c what if someone had used `$set` to add person properties - would they be considered identified from posthog.js?